backend/src/test/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/NormControllerTest.java

```diff
@@ -48,7 +49,7 @@ void itReturnsNorm() throws Exception {
       var norm =
           Norm.builder()
               .document(
-                  XmlMapper.toDocument(
+                  XmlProcessor.toDocument(
```

Comment:
Nitpicky thing: could you put a rename like XmlMapper -> XmlProcessor into a separate commit next time, that would make reading the diff a bit less noisy ^^